src/garima_s/ReverseArray.java

```diff
@@ -0,0 +1,18 @@
+package garima_s;
+
+public class ReverseArray {
+	
+	void printReverseArray(int[] input) {
+		int temp = 0;
+		System.out.print("Reverse Array : ");
+		for (int index = (input.length - 1); index >= 0; index--) {
+			temp = input[index];
```

Comment:
No need of temp variable,  directly print input[index]

Reply:
Updated